src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
 // write code here
+const tree = document.querySelector('.tree');
+
+(function wrapHeaderTextInSpan() {
+  const listOfLi = [...document.querySelectorAll('li')];
```

Comment:
Consider checking if there are any `<li>` elements before proceeding with the rest of the function to avoid unnecessary operations if the list is empty.